lib/utils/constants.dart

```diff
@@ -0,0 +1,17 @@
+abstract class Constants {
+  static const int backgroundColor255 = 255;
+  static const int backgroundColor0 = 0;
+  static const int backgroundColor217 = 217;
+  static const int bodyCrossAxisCount = 2;
+  static const double bodyPadding = 25.0;
+  static const double bodyMainAxisSpacing = 15.0;
+  static const double bodyCrossAxisSpacing = 40.0;
+  static const int bodyChildren = 6;
+  static const double bodyBorderRadius = 25.0;
+  static const double bottomNavigationBarChildren = 25.0;
+
+  static const int TextStyle189 = 189;
+  static const int TextStyle56 = 56;
+  static const int TextStyle27 = 27;
+  static const double TextStyle150_0 = 2;
```

Comment:
As i mentioned above, don't use the value that the constant will take as part of constant name.

Reply:
Remember that we use lowerCamelCase to name constants